docs/src/main/sphinx/release/release-363.md

```diff
@@ -0,0 +1,67 @@
+# Release 363 (5 Oct 2021)
+
+## General
+
+* Add `http` event listener implementation which sends JSON serialized events to a remote HTTP endpoint. ({issue}`8821`)
+* Improve performance of queries that group by `bigint` columns. ({issue}`9510`)
+* Reduce query memory usage when spill occurs. ({issue}`9270`, {issue}`9275`)
+* Reduce join memory usage. ({issue}`9327`)
```

Comment:
That sounds weird ..  isnt it more correct to say

Reduce memory usage for processing JOIN clauses

or something similar